sql/catalyst/benchmarks/CalendarIntervalBenchmark-results.txt

```diff
@@ -0,0 +1,11 @@
+================================================================================================
+interval
+================================================================================================
+
+Java HotSpot(TM) 64-Bit Server VM 1.8.0_311-b11 on Mac OS X 10.16
+Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
+CalendarInterval For interval:            Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
```

Comment:
the name is weird, `CalendarInterval` should be sufficient.

Reply:
OK